test/e2e/infinispan/upgrade_operand_test.go

```diff
@@ -292,13 +292,13 @@ func TestSpecImage(t *testing.T) {
 
 	// Create Infinispan Cluster using the latest Operand version, but using the image of the preceding Operand
 	// Ensures that creating a cluster with an initial spec.image value does not cause infinite StatefulSet updates
-	versionManager := tutils.VersionManager()
-	operand := versionManager.Operands[len(versionManager.Operands)-2]
+	imageOperand, versionOperand := specImageOperands()
```

Comment:
The comment needs to be updated to reflect that we're not using the latest operand versions, but rather two operands from the same major.minor stream.